assets/src/edit-story/components/panels/animation/effectPanel.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { ANIMATION_EFFECTS } from '../../../../animation/constants';
+import { AnimationProps } from '../../../../animation/parts/types';
+import { GetAnimationEffectProps } from '../../../../animation/parts';
```

Comment:
You're importing from /animation/parts/types as well as animation/parts - could you combine these and import from /animations/parts?

Reply:
Oooo good catch!  Will fix 🎉